Shared/Features/Freaks/View/FreaksView.swift

```diff
@@ -7,16 +7,35 @@ struct FreaksView: View {
         let coloredNavAppearance = UINavigationBarAppearance()
         coloredNavAppearance.titleTextAttributes = [.foregroundColor: UIColor(Color("SecondaryColor"))]
         coloredNavAppearance.backgroundColor = UIColor(Color("AccentColor"))
-        
+
         UINavigationBar.appearance().standardAppearance = coloredNavAppearance
         UINavigationBar.appearance().scrollEdgeAppearance = coloredNavAppearance
     }
 
     var body: some View {
         NavigationView {
-            List(viewModel.freaks, id: \.firstName) { freak in
-                NavigationLink(destination: FreakDetailsView(viewModel: FreakDetailsViewModel(freak: freak))) {
-                    ListItemView(viewModel: ListItemViewModel(freak: freak))
+            VStack(spacing: 0) {
+                HStack {
+                    Spacer()
+                    Button("Skills") {
+                        print("skills")
+                    }
+                    Spacer()
+                    Button("Projects") {
+                        print("projects")
+                    }
+                    Spacer()
+                }
+                .frame(height: 60)
```

Comment:
Applying this setup here, I think the buttons are very small, even the background color is correct. We should make sure the buttons are half the screen and 60 pixels in height. 
![Simulator Screen Shot - iPhone 12 Pro Max - 2021-07-22 at 16 55 10](https://user-images.githubusercontent.com/481425/126651122-adce3d17-6d8a-419b-8d6b-7655bc02f17e.png).

Reply:
https://www.hackingwithswift.com/quick-start/swiftui/how-to-provide-relative-sizes-using-geometryreader - I think using GeometryReader may help